### PR TITLE
web: i18n for download links

### PR DESCRIPTION
**Pull request by Ethanol48**
Created a getter function for  to reflect i18n and small change download page to add a word to i18n

### CODE REVIEW

**Comment by Ethanol48** [comment]
Fixed behavior mentioned in #1275

File: apps/daimo-web/src/app/download/other/page.tsx [hunk 1]
```diff
@@ -57,7 +58,7 @@ export default function DownloadPage() {
           </div>
           <div className="flex flex-row w-1/6 justify-center">
             <div className="h-[350px] flex flex-row items-center justify-center">
-              <p className="text-xl font-semibold text-grayMid">OR</p>
+              <p className="text-xl font-semibold text-grayMid">{i18.OR()}</p>
```
Comment (dcposch) [inline apps/daimo-web/src/app/download/other/page.tsx:61]: 👍

File: apps/daimo-web/src/utils/platform.ts [hunk 2]
```diff
@@ -12,6 +13,7 @@ export function detectPlatform(ua: string): PlatformType {
   return "other";
 }
 
+// necesary for some routes
```
Comment (dcposch) [inline apps/daimo-web/src/utils/platform.ts:16]: I think we can delete downloadMetadata > use `getDownloadMetadata` everywhere

**Review by dcposch — APPROVED** [review]
Otherwise LGreatTM

File: apps/daimo-web/src/utils/platform.ts (hunk 2 above)
Comment (Ethanol48) [reply apps/daimo-web/src/utils/platform.ts:16]: I'm not sure how to do that at the moment, there's this file `route.tsx` in the path /download that detectes the platform, I don't think that the hook `useI18n()` can be used, and I didn't want to over complicate the code

**Comment by Ethanol48** [comment]
I made the change to only use the function

**Pull request merged**